tools/gluten-it/common/src/main/scala/org/apache/gluten/integration/Suite.scala

```diff
@@ -195,6 +196,10 @@ abstract class Suite(
 
   private[integration] def allQueryIds(): Array[String]
 
+  protected[integration] def typeModifiers(): List[TypeModifier] = {
+    if (decimalAsDouble) List(new DecimalTypeModifier()) else List()
+  }
+
```

Comment:
nit: Could make it `private[integration]` and move it to line 189. It can still be overridden without `protected` since it's not final.